webpack.config.js

```diff
@@ -38,14 +35,16 @@ function createConfig(isDebug) {
   if (isDebug) {
     config.debug = true;
   } else {
-    config.output.filename = 'enigma.min.js';
-    config.plugins.push(new Webpack.optimize.UglifyJsPlugin());
+    config.plugins.push(new Webpack.optimize.UglifyJsPlugin({
+      compress: {
+        warnings: false,
+      },
+    }));
+
+    config.devtool = null;
```

Comment:
I'd rather keep the `.map` file even for production builds, please revert this line.